src/components/__tests__/ChatKeyboard.test.tsx

```diff
@@ -0,0 +1,39 @@
+import { screen, render, fireEvent, waitFor } from '@testing-library/react'
+import ChatKeyboard from 'components/ChatKeyboard'
+
+describe('<ChatMessage />', () => {
+  it('should render correctly', async () => {
+    render(<ChatKeyboard />)
```

Comment:
Dúvida, isso vale para todos os testes, esse render não deveria ser o customizado?

Reply:
So precisar ser o render personalizado, quando for testar estilos.

Reply:
Entendi, boa